test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -237,6 +453,13 @@ func testSecondaryNetwork(t *testing.T, networkType string, pods []*testPodInfo)
 			t.Fatalf("Error when pinging between interfaces: %v", err)
 		}
 	})
+
+	t.Run("testPauseAgentDeletePodRestartAgent", func(t *testing.T) {
+		err := testData.testPauseAgentDeletePodRestartAgent(pods[0], defaultTimeout)
+		if err != nil {
+			t.Fatalf("Error when restarting antrea agent: %v", err)
+		}
+	})
```

Comment:
Could we add a test to check after agent restart for the following:
1) a Pod that is not deleted should still have its secondary interface OVS port
2) delete the Pod
3) check OVS port is deleted and IP is released to the IPPool